src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -169,7 +169,7 @@ module.exports = function<T, P, I, C>(config : HostConfig<T, P, I, C>) {
         // it until the next tick.
         workInProgress.child = current.child;
         reuseChildren(workInProgress, workInProgress.child);
-        if (workInProgress.pendingWorkPriority <= priorityLevel) {
+        if (workInProgress.pendingWorkPriority !== NoWork && workInProgress.pendingWorkPriority <= priorityLevel) {
```

Comment:
The problem is that without this technique we don't have a way to go to arbitrary lower priorities. What number do we choose that is a fast path since `Infinity` is a slow path in some platforms?